CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ StackOverflow is a much better place to ask questions since:
 * questions and answers stay available for public viewing so your question / answer might help someone else
 * SO voting system assures that the best answers are prominently visible.
 
-To save your and our time we will be systematically closing all the issues that are request for general support and redirecting people to StackOverflow. 
+To save your and our time we will be systematically closing all the issues that are requests for general support and redirecting people to StackOverflow.
```

Comment:
Yea, I was going back and forth on that and was wondering if it would come across as harsh.  You just never know what mood someone is going to be in at any given time.